puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katello/nightly.groovy

```diff
@@ -59,7 +59,7 @@ pipeline {
                     post {
                         always {
                             script {
-                                set_job_build_description()
+                                set_job_build_description("foreman-katello-nightly-test")
```

Comment:
Maybe set this as a environment variable to reference in the steps and post given we now have it in 3 places?

Reply:
like this?